bin/aws-cli-mfa-fish

```diff
@@ -0,0 +1,182 @@
+#!/usr/bin/env fish
+
+# you MUST source this file to get the exports in your shell!
+if test "$_" = source || test -n "$IS_SOURCED"
+    set SOURCED 1
+else
+    set SOURCED 0
+end
+
+set -l REQUIREMENTS python3 aws jq
+
+for requirement in $REQUIREMENTS
+    if not command -v $requirement > /dev/null 2>&1
+        echo "$requirement could not be found!"
+        exit 0
+    end
+end
+
+set -l PYCODE "
```

Comment:
The difference between heredocs and fish multiline strings was tricky, so we have to pre-process the python script to escape any quotes that conflict with the top level.

Reply:
The other files in `/bin` also have the python code embedded

Reply:
Ohhhhhh I completely ignored that this is the bin version of the script!